GUIDE.md

```diff
@@ -1522,34 +1601,43 @@ built packages, meaning that subsequent builds will be much faster.
 
 Two notes for future improvement:
 
-* One Travis whitelists the stack .deb files, we'll be able to simply include stack in the `addons` section, and automatically use the newest version of stack, avoiding that complicated `before_install` section
-* Starting with stack-0.1.4.0, there are improvements to the test command, so that the entire script section can be `stack --no-terminal --install-ghc test`
+* One Travis whitelists the stack .deb files, we'll be able to simply include
```

Comment:
Yeah, I had *no* clue what "One Travis" was, so I just left it alone :P will fix